source/includes/_endpoint-dataset.md

```diff
@@ -486,6 +486,16 @@ in the API.
 
 Categorical variables will be exported with their name instead of their value.
 
```

Comment:
RE: views/value 

I believe views is correct, a shoji view can have subviews, in this case csv and spss, but maybe fumanchu can verify.  This part of the doc was existing.